RadixWallet/Features/SettingsFeature/AccountSecurity/AccountSecurity+Reducer.swift

```diff
@@ -15,8 +17,13 @@ public struct AccountSecurity: Sendable, FeatureReducer {
 		public var canImportOlympiaWallet = false
 
 		public static let importOlympia = Self(destination: .importOlympiaWallet(.init()))
+		public var deepLinkToDisplayMnemonics: Bool
```

Comment:
uuuuuuuhmm 😩😩😩😩😩 this is terrible... but I cant come up with anything better :/ ideas?